src/sdk/flyover.ts

```diff
@@ -452,6 +453,15 @@ export class Flyover implements Bridge {
     }
   }
 
+  /**
```

Comment:
Pls also clarify the connections requited (if rsk, btc, or both) also for the method that executes the register pegin